mcs/class/corlib/System.Reflection.Emit/AssemblyBuilder.cs

```diff
@@ -1062,7 +1062,7 @@ internal Type MakeGenericType (Type gtd, Type[] typeArguments)
 		{
 			return new TypeBuilderInstantiation (gtd, typeArguments);
 		}
-
+/*
```

Comment:
should be removed if we no longer need it

Reply:
Will clean that up